presto-main/src/test/java/io/prestosql/util/TestTimeZoneUtils.java

```diff
@@ -35,7 +35,6 @@ public void test()
     {
         TimeZoneKey.getTimeZoneKey("GMT-13:00");
 
-        TreeSet<String> jodaZones = new TreeSet<>(DateTimeZone.getAvailableIDs());
```

Comment:
This is not related to adding a new zone, it's an unrelated cleanup.
Normally you'd extract this to a separate commit (within this PR), but there are other things to clean up in this class, so I put up https://github.com/prestosql/presto/pull/1249.
Thus, you can drop this change from here.

Reply:
@findepi Sure, removed from this PR. Originally, forgot to delete that line in https://github.com/prestosql/presto/pull/1222